library/item/item_content_registry/src/main/java/org/quiltmc/qsl/item/content/registry/impl/ItemContentRegistriesClientInitializer.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2022 QuiltMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.quiltmc.qsl.item.content.registry.impl;
+
+import org.quiltmc.loader.api.ModContainer;
+import org.quiltmc.loader.api.QuiltLoader;
+import org.quiltmc.qsl.base.api.entrypoint.client.ClientModInitializer;
+import org.quiltmc.qsl.item.content.registry.api.ItemContentRegistries;
+import org.quiltmc.qsl.tooltip.api.client.ItemTooltipCallback;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import net.minecraft.item.Item;
+import net.minecraft.text.Text;
+
+
+public class ItemContentRegistriesClientInitializer implements ClientModInitializer {
+	public static final Logger LOGGER = LoggerFactory.getLogger("ItemContentRegistriesClientInitializer");
+
+	public static final String ENABLE_TOOLTIP_DEBUG = "quilt.item.content_registry.enable_tooltip_debug";
```

Comment:
It's a bit redundant, but yeah, the module's name is Item Content Registry
```suggestion
	public static final String ENABLE_TOOLTIP_DEBUG = "quilt.item.item_content_registry.enable_tooltip_debug";
```